include/swift/AST/PrintOptions.h

```diff
@@ -425,6 +425,7 @@ struct PrintOptions {
     result.AbstractAccessors = false;
     result.PrintForSIL = true;
     result.PrintInSILBody = true;
+    result.PreferTypeRepr = false;
```

Comment:
Theoretically this would change behaviour for non-VarDecl type printing though right?  For swift-ide-test I think that's fine, but I'd rather @slavapestov or @jckarter commented on any change for SIL printing.